pkg/codegen/python/gen.go

```diff
@@ -1215,8 +1216,34 @@ func sanitizePackageDescription(description string) string {
 	return ""
 }
 
+func genPulumiPluginFile(pkg *schema.Package) (string, error) {
+	type pulumiPlugin struct {
```

Comment:
Can we define and version this serialization format in a single place?  I see this is re-defined in the Python language host implementation.